ingestion/src/metadata/ingestion/source/dynamodb.py

```diff
@@ -77,18 +78,22 @@ def ingest_tables(self, next_tables_token=None) -> Iterable[OMetaDatabaseAndTabl
         tables = list(self.dynamodb.tables.all())
         for table in tables:
             try:
-                if not self.config.table_filter_pattern.included(table.name):
```

Comment:
The filtering logic is now centralised https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/utils/filters.py let's see if we can use that or update it if required 🙏

Reply:
yes I'll check it